ios/GoogleService-Info.plist

```diff
@@ -1,40 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-  <dict>
```

Comment:
I'm not sure we want to keep this file in the repo, we have an example.GoogleService-Info.plist in the ios folder that a script that sets it up is copying on CI. I'm creating a ticket to update the docs with this info.